src/components/Footer.vue

```diff
@@ -29,7 +29,7 @@ export default {
   components: { ColorSchemeToggle, LocaleSelector },
   computed: {
     enablei18n: () => (
-      getSetting(['features', 'docs', 'i18n', 'enable'], false)
+      getSetting(['features', 'docs', 'i18n', 'enable'], true)
```

Comment:
Would we make it optional to disable or did this slip away?

Reply:
Correct one has `false` as fallback :) fixed